src/sentry/south_migrations/0416_auto__del_field_identityprovider_organization__add_field_identityprovi.py

```diff
@@ -0,0 +1,1160 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.delete_foreign_key(u'sentry_identityprovider', 'organization_id')
+
+    def backwards(self, orm):
+        sql = db.foreign_key_sql(
```

Comment:
Since you wrote this, I'd argue that we don't need this `backwards`. We will never roll back in production, and even attempting to roll back in this case would be problematic. Since we're breaking the FK constraint, it's likely that there'd be a row that violates, causing the backwards to fail anyways.